sale_double_validation/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Consider generating a nice README with https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py

Reply:
Please consider merging it as the PR was made  on 2017/04/05.
I'm unwilling to change this.